lib/testing/test_transaction.ex

```diff
@@ -0,0 +1,84 @@
+defmodule Braintree.Testing.TestTransaction do
```

Comment:
In fact, the entire module can be conditionally defined based on the current environment.